acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/KeyLoadAndSignAcceptanceTest.java

```diff
@@ -108,12 +107,35 @@ public void receiveA400IfJsonBodyIsMalformed() throws Exception {
     assertThat(response.getStatusCode()).isEqualTo(400);
   }
 
+  @Test
+  public void signsDataContainingUnknownFields() throws Exception {
```

Comment:
nit: maybe unusedFieldsInRequest do not affect signing operation?

Reply:
done